DNN Platform/Modules/Dnn.Modules.DynamicContentManager/ClientScripts/Manager.js

```diff
@@ -1,4 +1,18 @@
-﻿function Manager($, ko, codeEditor, settings, resx){
+﻿ko.bindingHandlers.truncatedText = {
```

Comment:
Hi @mgatmaytan, I would try to avoid knockout binding handlers any time that we can obtain the same result with just css code. Have a look at this example https://jsfiddle.net/w3avhaon/. Let me know if you have any questions.